WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/AddressSelectorCommand.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+import Yosemite
+import SwiftUI
+
+protocol AddressSelectorCommandProtocol {
```

Comment:
I think we are not selecting addresses here, but more likely "areas". What about if we change the Address mentions to something like `AreaSelectorCommandProtocol`, or `AreaSelectorCommand`?  If we do that, we would have to change the comments and other references as well.

Reply:
Definitely, changed as well on https://github.com/woocommerce/woocommerce-ios/pull/7713/commits/4caaa7c8fb6086c837c992223170554cd560a2c1